src/components/Dashboard/StaffNotation.vue

```diff
@@ -154,15 +174,26 @@
 
 <script lang="ts">
 import Vue from 'vue'
-import { Component, Prop, Emit } from 'vue-property-decorator'
-import { Getter } from 'vuex-class'
+import { Component, Emit, Prop } from 'vue-property-decorator'
 import { navigate } from '@/utils'
-import { DissolutionTypes, DissolutionNames, FilingTypes, FilingNames } from '@/enums'
+import { ApplicationTypes, DissolutionNames, DissolutionTypes, FilingNames, FilingTypes, RestorationTypes } from '@/enums'
 import { AddStaffNotationDialog } from '@/components/dialogs'
 import { FilingMixin } from '@/mixins'
 import { LegalServices } from '@/services'
+import { mapGetters } from 'vuex'
 
 @Component({
+  computed: {
+    ...mapGetters([
+      'getIdentifier',
+      'isAdminFreeze',
+      'isBenBcCccUlc',
+      'isCoop',
+      'isEntityInLimitedRestoration',
+      'isFirm',
+      'isHistorical'
+    ])
+  },
```

Comment:
I'm OK with this format (instead of using `@Getter` decorators. I think this is more in line with [Vue3 Composition API](https://vuejs.org/guide/extras/composition-api-faq.html) anyway.